### DISCUSSION TITLE
External keyboard key assignments - Map Orientation (Android Version)

### DISCUSSION THREAD

**Discussion by fjb2020** [osmandapp/OsmAnd#20707] [General]
I use an external input device on my motorbike (JaxeADV BarButtons) and on earlier versions of OsmAnd+ was on my Android phone was able to cycle through the Map orientation options via a simulated 'D' keypress. The current releases of OsmAnd on Android offer far more customisation of how external keyboards can control the app but I can't find how to change the map orientation other than via tapping the compass icon on the screen.

I've looked at the options around Key assignments and actions but can't see any way of changing map orientation via an external keyboard anymore on the Android version - it still works on the iOS version which doesn't have the more sophisticated external keyboard options.

Am I missing something or is this feature no longer available?

Thanks
Fiona

**Comment by dmpr0** [comment]
Hi!
Yes, we have missed few actions. Now all of that actions are part of the Quick action list, so they can be used for External input devices or as a custom button on the map. 
Missed actions were added https://github.com/osmandapp/OsmAnd/issues/20636. Currently, it is available in nightly versions.

**Reply by fjb2020** [reply]
Ahh ok, thanks - sorry I missed that update
